src/common/Kubernetes/K8sClientFactory.cs

```diff
@@ -109,7 +109,14 @@ public IKubernetes CreateFromKubeConfigStream(Stream kubeConfigStream)
         /// </summary>
         public IKubernetes CreateFromKubeConfig(K8SConfiguration kubeConfig)
         {
-            var config = KubernetesClientConfiguration.BuildConfigFromConfigObject(kubeConfig);
+            KubernetesClientConfiguration config = null;
+            if (_environmentVariables.KubectlProxy != null)

```

Comment:
💡 Idea and thoughts: we can avoid this for `cyclomatic complexity` to just this below: (Please feel free to choose whatever style you think is good. 

```*.cs
            if (_environmentVariables.KubectlProxy != null)
            {
                config = new KubernetesClientConfiguration { Host = _environmentVariables.KubectlProxy };
                return new k8s.Kubernetes(config);
           }
           
           config = KubernetesClientConfiguration.BuildConfigFromConfigObject(kubeConfig);
           return new k8s.Kubernetes(config);
```